packages/simcore-sdk/src/simcore_sdk/node_ports_common/aws_s3_cli_utils.py

```diff
@@ -0,0 +1,42 @@
+import logging
+import re
+
+from pydantic import ByteSize, parse_obj_as
+from servicelib.logging_utils import log_catch
+from servicelib.progress_bar import ProgressBarData
+
+from .r_clone_utils import BaseLogParser
```

Comment:
why is this one depending on rclone? should this BaseLogParser not go into a common syn_utils.py file instead?